kuma/static/styles/components/home/column-callout.scss

```diff
@@ -135,3 +105,127 @@ Colours and icons
         background-position: 0 -600px;
     }
 }
+
+/*
+Promos
+-------------------------------------------------------------- */
+.callout-saucelabs {
+    background-color: #82D1F5;
+
+    a {
+        color: #000 !important;
+        font-weight: bold;
+
+        &:after {
+            @include bidi-style(right, 20px, left, 20px);
+            background-image: url($path-to-embedded-images + 'promos/sauce-labs-home.png');
+            height: 140px;
+            top: 15px;
+            width: 129px;
+        }
+
+        &:before {
+            content: none;
+            display: none;
+        }
+
+        button {
+            @include bidi-style(text-align, left, text-align, right);
+            @include set-font-size(14px);
+            background: #82D1F5;
+            border: 2px solid #fff;
+            border-radius: 0;
+            box-shadow: none;
+            color: #000;
+            display: inline-block;
+            font-weight: bold;
+            margin-top: 14px;
+            padding: 9px 18px;
+            text-transform: none;
+        }
+    }
+}
+
+/*
+Media queries
+-------------------------------------------------------------- */
+@media (-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 3/2), (min-resolution: 1.5dppx) {
+    .callout-saucelabs a:after {
+        background-image: url($path-to-embedded-images + 'promos/sauce-labs-home-high-res.png');
+        background-size: 129px 140px;
+    }
+}
+
+@media #{$media-query-small-desktop} {
+    .callout-saucelabs a:after {
+        @include bidi-style(right, -20px, right, auto);
+        @include bidi-style(left, auto, left, -20px);
+    }
+}
+
+@media #{$media-query-tablet} {
+    .column-callout a:after {
+        top: $grid-spacing;
+        @include bidi-style(right, $grid-spacing, left, auto);
+    }
+
+    .callout-saucelabs {
+        a {
+            @include set-font-size(14px);
+
+            &:after {
+                @include bidi-style(right, -70px, right, auto);
+                @include bidi-style(left, auto, left, -70px);
+            }
+        }
+
+        span {
+            width: 90%;
+        }
+    }
+}
+
+@media #{$media-query-mobile} {
+
+    .column-callout {
+        margin-bottom: $grid-spacing;
+    }
+
+    .column-callout a:after {
+        top: $callout-icon-spacing;
+        @include bidi-style(right, $callout-icon-spacing + $grid-spacing, left, auto);
+    }
+
+    .callout-saucelabs a {
```

Comment:
Nit: On line 172 above, this is written as:

```
.callout-saucelabs {
  a {
```

The resulting CSS will be a little different i.e.

```
.callout-saucelabs a { ... };
.callout-saucelabs span { ... };
```
 vs.

```
.callout-saucelabs a { ... };
.callout-saucelabs a span { ... };
```
 Same end result, just nit picking ;)